.github/workflows/pdk-basic.yml

```diff
@@ -0,0 +1,52 @@
+# This is a basic workflow to help you get started with Actions
+
+name: PDK basic
+
+# Controls when the workflow will run
+on:
+  workflow_call:
+    inputs:
+      container_image:
+        description: Image to use when running tests
+        default: 'puppet/puppet-dev-tools:2022-03-28-92c7176'
+        required: false
+        type: string
+
+
+env:
+  GITHUB_USER: ${{ secrets.PUPPET_REPO_USER }}
+  GITHUB_TOKEN: ${{ secrets.PUPPET_REPO_USER_TOKEN }}
```

Comment:
Does this work? I thought you need to specify the secrets as inputs

Reply:
This was removed as it is currently not being used.